### PR TITLE
Toolbar and headlines creation

### PR DESCRIPTION
**Pull request by pirc25**
### Toolbar and headline creation
I used  Figma Template 2 to create a portfolio mobile skeleton. 
In this pull request the changes are:

- Add Toolbar for mobile
- Add Headline for mobile
- Edit Readme.MD with project information.
- Add HTML&CSS linters

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+- GitHub: [@pirc25](https://github.com/pirc25)
+- Twitter: [@RichardBurneo](https://twitter.com/RichardBurneo)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/richard-burneo-aguilera-1277aa203/)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
```
Comment (Temmarie) [inline README.md]: - [x] Kindly remove this second author information since it's only one person that worked on it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+</head>
+
+<body>
+  <header id="brand" class="toolbar">
+    <div class="my-logo">
+      Richard Burneo
+    </div>
+    <nav class="menu-botton">
+      <button><i class="fa fa-bars fa-spin"></i></button>
+    </nav>
```
Comment (Temmarie) [inline index.html]: - [x] Kindly remove the `fa-spin` class from the menu bar icon, as it doesn't follow the template design. While it's good for learning animation, it doesn't fit the portfolio's aesthetics. 
- [x] It'll be good if you could add a `padding:10px` to the header as well, to follow the template design you've picked. Currently, the spacing is not enough.
- [x] It's also a good idea to style the menu bars to fit in with the provided template design or remove the menu bars from the button tag as they currently are taking on features of a button.

 _See images below_

**Original** | **Your project**
  ----------- | -----------
  image | image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+        </li>
+        <li>
+          <a href="#" class="Instagram">
+            <i class="fa-brands fa-instagram"></i>
+          </a>
+        </li>
+
+
+      </ul>
+    </div>
```
Comment (Temmarie) [inline index.html]: - [x] Kindly add the proper background image (decorations are always background-images, not `img` tags). 
- [x] KIndly style the social media icons to follow the design template, they are currently still taking on the `a` tag properties.

 _See images below_

**Original** | **Your project**
  ----------- | -----------
  image | image

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @pirc25  ! :wave: :wave:

#### You have done really well in implementing the requirements.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 
- - -

### **Recommended** ♻️
- [ ] You've done really well so far, however, there are some parts of your project that doesn't follow the provided Figma design template.

_See the inline comments below for more required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A
_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you have a limited amout reviews as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

File: README.md (hunk 1 above)
Comment (pirc25) [reply README.md]: Thank you for your review. This is done.

File: index.html (hunk 2 above)
Comment (pirc25) [reply index.html]: Thank you for the review.
I have finished:
- Change the menu bar icon.
- Add padding:10px  to the header:
- Edit menu bars similar to the template design.

File: index.html (hunk 3 above)
Comment (pirc25) [reply index.html]: I did it, but I have problems with the size background image.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,106 @@
+@import "https://fonts.googleapis.com/css?family=Crete Round|Inter";
+
+header {
+  padding: 12px;
+}
```
Comment (AmaduKamara) [inline style.css:10]: Good job working on the padding here.

[ Optional Changes ] All changes marked optional won't stop your program from being approved but will help make your program better.

- [x] Kindly add the code below above the `header` style to reset all styles on the entire page to take off the white margin on all corners of the website in order for your design to match the Figma template. 
```mdx 
* {
  margin: 0;
  padding: 0;
}
```

**Your design**
image

**The Figma Design**
image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+        </li>
+        <li>
+          <i class="fa-brands fa-twitter"></i>
+        </li>
+        <li>
+          <i class="fa-solid fa-m"></i>
+        </li>
+      </ul>
+    </div>
+  </section>
```
Comment (AmaduKamara) [inline index.html:52]: [Optiona Changes ]  Kindly style your social media icons in this section to be on the left. If you take a look at the design, the icons are placed on the left. Please fix that as

[Optiona Changes ]  Also please add some spacing from the top of the header to the heading text. The previous reviewer did not spot that, but it's required your style matches the design in the Figma template. Please fix that 👍 

**Your Design**

image

**The Figma Design**
image

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @pirc25,

Good job working on the changes requested by the previous reviewer. 👍 

ClappingLeonardoDicaprioGIF


Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

## Recomendations

- [x] Kindly work on the optional changes I made in the comments in order to save the hassles on your next milestone. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**